content/tutorial-multisign.md

```diff
@@ -0,0 +1,603 @@
+
+Normally, an account can own many objects of different types (such as trust lines and offers). If you funded a new address for this tutorial, the SignerList is the only object in the response.
+
+    $ rippled account_objects rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC validated
+    Loading: "/home/mduo13/.config/ripple/rippled.cfg"
+    Connecting to 127.0.0.1:5005
+    {
+       "result" : {
+          "account" : "rEuLyBCvcw4CFmzv8RepSiAoNgF8tTGJQC",
+          "account_objects" : [
+             {
+                "Flags" : 0,
+                "LedgerEntryType" : "SignerList",
+                "OwnerNode" : "0000000000000000",
+                "PreviousTxnID" : "8FDC18960455C196A8C4DE0D24799209A21F4A17E32102B5162BD79466B90222",
+                "PreviousTxnLgrSeq" : 5,
+                "SignerEntries" : [
+                   {
+                      "SignerEntry" : {
+                         "Account" : "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+                         "SignerWeight" : 2
+                      }
+                   },
+                   {
+                      "SignerEntry" : {
+                         "Account" : "raKEEVSGnKSD9Zyvxu4z6Pqpm4ABH8FS6n",
+                         "SignerWeight" : 1
+                      }
+                   },
+                   {
+                      "SignerEntry" : {
+                         "Account" : "rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v",
+                         "SignerWeight" : 1
+                      }
+                   }
+                ],
+                "SignerListID" : 0,
+                "SignerQuorum" : 3,
+                "index" : "79FD203E4DDDF2EA78B798C963487120C048C78652A28682425E47C96D016F92"
+             }
+          ],
+          "ledger_hash" : "56E81069F06492FB410A70218C08169BE3AB3CFD5AEA20E999662D81DC361D9F",
+          "ledger_index" : 5,
+          "status" : "success",
+          "validated" : true
+       }
+    }
+
+If the SignerList is present with the expected contents, then your address is ready to multi-sign.
+
+### 6. Further steps ###
+
+At this point, your address is ready to [send a multi-signed transaction](#sending-a-multi-signed-transaction). You may also want to:
+
+* Disable the address's master key by sending an [AccountSet transaction](reference-transaction-format.html#accountset) using the `asfDisableMaster` flag.
+* Remove the address's regular key (if you previously set one) by sending a [SetRegularKey transaction](reference-transaction-format.html#setregularkey).
+
+
+Sending a Multi-Signed Transaction
+----------------------------------
+
+Before you can multi-sign a transaction, first check that [multi-sign is available](#availability-of-multi-signing) and [set up multi-signing](#set-up-multi-sign) for your address. The following procedure demonstrates how to create, sign, and submit a multi-signed transaction.
```

Comment:
Good point. I moved the part about checking whether multi-signing is available to the Setting Up Multi-Signing section.